src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -2419,18 +2447,26 @@ private Position InternalAnchorPoint
             get
             {
                 Position temp = new Position(0.0f, 0.0f, 0.0f);
+#if NUI_PROPERTY_CHANGE_1
```

Comment:
same as above

Reply:
I check that we already have `AnchorPoint` relative API internal. and they use Vector3.

`ParentOrigin` use `Position`type so I also change it use `Position` type, and also use `ViewInternal.cs` API.